### PR TITLE
Add July 2023 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--1981.org.readthedocs.build/

### CODE REVIEW

File: docs/topics.rst [hunk 1]
```diff
@@ -171,6 +171,7 @@ Other specific doc types
 
 Including how-tos, reference docs, tutorials, and books.
 
+- |:newspaper:| `Documenting YAML Files </blog/newsletter-may-2023/#documenting-yaml-files>`__
```
Comment (plaindocs) [inline docs/topics.rst]: These should be s/may/july/ right?

Comment (CollierCZ) [reply docs/topics.rst]: Yep. Thanks!

Comment (CollierCZ) [reply docs/topics.rst]: /
```suggestion
- |:newspaper:| `Documenting YAML Files </blog/newsletter-july-2023/#documenting-yaml-files>`__
```

File: docs/topics.rst [hunk 2]
```diff
@@ -285,6 +286,7 @@ Hiring
 Getting hired
 ~~~~~~~~~~~~~
 
+- |:newspaper:| `Setting Up for Success as a Neurodivergent Person </blog/newsletter-may-2023/#setting=up-for-success-as-a-eurodivergent-person>`__
```
Comment (CollierCZ) [inline docs/topics.rst]:
```suggestion
- |:newspaper:| `Setting Up for Success as a Neurodivergent Person </blog/newsletter-july-2023/#setting-up-for-success-as-a-eurodivergent-person>`__
```

File: docs/topics.rst [hunk 3]
```diff
@@ -719,6 +721,7 @@ Working with other roles
 
 Including product managers, higher-ups, etc.
 
+- |:newspaper:| `Getting Info from Others </blog/newsletter-may-2023/#getting-info-from-thers>`__
```
Comment (CollierCZ) [inline docs/topics.rst]:
```suggestion
- |:newspaper:| `Getting Info from Others </blog/newsletter-july-2023/#getting-info-from-others>`__
```

File: docs/topics.rst [hunk 4]
```diff
@@ -751,6 +754,7 @@ Planning and how we work
 Prioritisation
 ~~~~~~~~~~~~~~
 
+- |:newspaper:| `Pushing Back When You're Overloaded </blog/newsletter-july-2023/#pushing-back-when-you-re-verloaded>`__
```
Comment (CollierCZ) [inline docs/topics.rst]:
```suggestion
- |:newspaper:| `Pushing Back When You're Overloaded </blog/newsletter-july-2023/#pushing-back-when-you-re-overloaded>`__
```

File: docs/topics.rst [hunk 5]
```diff
@@ -285,6 +286,7 @@ Hiring
 Getting hired
 ~~~~~~~~~~~~~
 
+- |:newspaper:| `Setting Up for Success as a Neurodivergent Person </blog/newsletter-july-2023/#setting-up-for-success-as-a-eurodivergent-person>`__
```
Comment (CollierCZ) [inline docs/topics.rst]:
```suggestion
- |:newspaper:| `Setting Up for Success as a Neurodivergent Person </blog/newsletter-july-2023/#setting-up-for-success-as-a-neurodivergent-person>`__
```

File: docs/blog/newsletter-july-2023.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
+.. post:: July 07, 2023
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – July 2023
+#####################################
+
+Aloha! There's lots to cover this month, so let's dive in.
+
+Who makes up the Write the Docs community? What proportion support pay transparency?  Where are salaries highest? How many have started a new job in the past 12 months? All these questions and more are answered in the `2022 Write the Docs Salary Survey results <https://writethedocs.org/surveys>`__. Stay tuned: the survey for 2023 will be opening in early September.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-july-2023.rst]:
```suggestion
Who makes up the Write the Docs community? What proportion support pay transparency? Where are salaries highest? How many have started a new job in the past 12 months? All these questions and more are answered in the `2022 Write the Docs Salary Survey results <https://writethedocs.org/surveys>`__. Stay tuned: the survey for 2023 will be opening in early September.
```

File: docs/blog/newsletter-july-2023.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+Getting Info from Others
+------------------------
+
+A recent thread resurrected an enduring issue with documentarians: How do we get the information we need from experts? The `May 2020 newsletter </blog/newsletter-may-2020/#getting-more-info-from-smes>`__ addressed this issue. The current discussion focused on how to ask questions of developers: what questions to ask and how to get the best responses for documentation. 
+
+People indicated that there are two levels of questions: 1) general, high-level questions; and 2) specific, detailed questions. Documentarians ask general questions to get an overall understanding of a product or domain. After we have the higher-level understanding, we can determine what detailed questions to ask. Specific questions fill in the gaps in our knowledge.
+
+For general questions, you can ask them of product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-july-2023.rst]: In the first sentence, maybe "For general questions, you can ask product managers..." (remove the `them of`)?

**Review by hillaryfraley — APPROVED** [review]

**Comment by plaindocs** [comment]
Can we hold off on merging this for a day or so.

File: docs/blog/newsletter-july-2023.rst (hunk 7 above)
Comment (CollierCZ) [reply docs/blog/newsletter-july-2023.rst]: Sounds better to me.

```suggestion
For general questions, you can ask product managers or customer support. Another potential resource could be any QA testers. We also have `learning resources on the topic </topics/#working-with-other-roles>`__ for interacting with people in other positions. If you're writing end-user documentation, this knowledge along with your personal experience of the product may be sufficient.
```

**Comment by CollierCZ** [comment]
> Can we hold off on merging this for a day or so.

We certainly can. What are we waiting for?

**Comment by plaindocs** [comment]
The other thing to wait for would be publishing the full Australia site, over in #1978
I'll approve that tonight (hopefully), and @swapnilogale can merge it in tomorrow morning (EU time).

**Comment by CollierCZ** [comment]
So should we merge or wait till next week, @plaindocs ?

**Comment by swapnilogale** [comment]
> So should we merge or wait till next week, @plaindocs ?

Australia website PR is ready for a final approval @plaindocs.

@CollierCZ depending on where we land with the AU website, can we add a section in the newsletter before it goes out?

**Comment by CollierCZ** [comment]
@swapnilogale , there's already a note about news coming soon. Can adjust it if the other PR gets merged today.

**Comment by swapnilogale** [comment]
> @swapnilogale , there's already a note about news coming soon. Can adjust it if the other PR gets merged today.

Yep, that's what I meant. Sorry for the confusion!

**Comment by plaindocs** [comment]
@CollierCZ I just approved https://github.com/writethedocs/www/pull/1978 do you want to tweak the Australia copy here?

File: docs/blog/newsletter-july-2023.rst [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+#####################################
+Write the Docs Newsletter – July 2023
+#####################################
+
+Aloha! There's lots to cover this month, so let's dive in.
+
+Who makes up the Write the Docs community? What proportion support pay transparency? Where are salaries highest? How many have started a new job in the past 12 months? All these questions and more are answered in the `2022 Write the Docs Salary Survey results <https://writethedocs.org/surveys>`__. Stay tuned: the survey for 2023 will be opening in early September.
+
+In conference news, we recently saw the announcement of the `speakers for the 2023 Atlantic conference </conf/atlantic/2023/news/announcing-speakers/>`__. So check out all the interesting things you can look forward to learning. And the `Australia conference </conf/australia/2023/news/welcome/>`__ is getting ready for the call for proposals and ticket sales so stand by for more about that.
```
Comment (CollierCZ) [inline docs/blog/newsletter-july-2023.rst]:
```suggestion
In conference news, we recently saw the announcement of the `speakers for the 2023 Atlantic conference </conf/atlantic/2023/news/announcing-speakers/>`__. So check out all the interesting things you can look forward to learning. And the Australia conference has released the `call for proposals </conf/australia/2023/cfp/>`__ and `ticket sales </conf/australia/2023/tickets/>`__ so get ready for good times in December.
```

**Comment by CollierCZ** [comment]
Copy is updated with links

**Comment by swapnilogale** [comment]
Looks good. Thanks for updating this promptly Aaron.

**Pull request merged**